build.sbt

```diff
@@ -1,19 +1,19 @@
 import Dependencies._
 
-lazy val defaultSettings = Seq(
-  name := "cromwell_pipeline",
-  version := "0.1",
-  scalaVersion := "2.12.9",
-  defaultScalacOptions)
+lazy val defaultSettings =
+  Seq(name := "cromwell_pipeline", version := "0.1", scalaVersion := "2.12.9", defaultScalacOptions)
 
-lazy val cromwell_pipeline = (project in file("."))
-    .settings(
-      defaultSettings, 
-      libraryDependencies ++= akkaDependencies ++ dbDependencies ++ testDependencies
-    )
+lazy val cromwell_pipeline = (project in file(".")).settings(
```

Comment:
If it is the result of formatting, personally I prefer to have `.settings` on the next line